src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/StringFWTest.java

```diff
@@ -30,16 +31,81 @@
 {
     private static final int LENGTH_SIZE = 1;
 
-    private final MutableDirectBuffer buffer = new UnsafeBuffer(allocateDirect(100))
+    private final MutableDirectBuffer buffer = new UnsafeBuffer(allocateDirect(100));
     {
-        {
-            // Make sure the code is not secretly relying upon memory being initialized to 0
-            setMemory(0, capacity(), (byte) 0xab);
-        }
-    };
+        buffer.setMemory(0, buffer.capacity(), (byte) 0xab);
+    }
+    MutableDirectBuffer expected = new UnsafeBuffer(allocateDirect(100));
+    {
+        expected.setMemory(0, expected.capacity(), (byte) 0xab);
+    }
```

Comment:
You'll need to add methods to verify setting string value to null. I added these in a PR which has now been merged to develop: see https://github.com/reaktivity/nukleus-maven-plugin/pull/56/files#diff-079c24d5437e326d56da05271d2f2a8b

If you do `git pull upstream develop` that will pull in the changes. You'll probably have to resolve some merge conflicts.